array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,22 @@
+/* exported takeRight */
+/*
+create a function that returns a new array containing the last x amount of elements in an array
+create a storage container
+if the count is higher than items in the string, return the og string
+iterate through each item that falls within x indexes of the final index, starting from the lowest index
+iterate until you reach the final index
+push each item to the new storage container as you iterate
+when all indexed have been searched, return the value of the container
+*/
+
+function takeRight(array, count) {
+  var container = [];
+  for (var i = 0; i < count; i++) {
+    if (count > array.length) {
+      return array;
```

Comment:
You should do this conditional above the loop, so it doesn't get checked over and over on each iteration of the loop.